lib/Elastica/Filter/GeoBoundingBox.php

```diff
@@ -39,15 +39,15 @@ public function __construct($key, array $coordinates)
      */
     public function addCoordinates($key, array $coordinates)
     {
-        if (!isset($coordinates[0]) || !isset($coordinates[1])) {
-            throw new InvalidException('expected $coordinates to be an array with two elements');
-        }
+        if (isset($coordinates[0], $coordinates[1])) {
```

Comment:
Before was:
1. Check as broken - throw
2. Return normal

After:
1. Check normal - return normal
2. Throw

Maybe is `as for me` hard reading `not`.
But return as `was` before/